ui/src/views/mint/github-step/steps/github-repository-selection/repository.tsx

```diff
@@ -28,14 +26,15 @@ export const Repository: React.FC<RepositoryProps> = ({
 
   return (
     <>
-      <RepoRow
-        onClick={handleSelectRepo}
-        repo={repository.name}
-        css={{ cursor: 'pointer' }}
-        button={
+      <RowData
+        leftIcon={<Icon name="github" css={{ fontSize: '$2xl', mr: '$2' }} />}

```

Comment:
I think you could set the Icon styles with the RowData styles declaration, this `css` is twiced in the usage.